SWIGGYDEMO_REACT/swiggy-react-app/src/components/Filters/components/favouriteCheckbox/FavouriteCheckboxContainer.js

```diff
@@ -1,10 +1,8 @@
-import React from "react";
-import FavouriteCheckbox from "./FavouriteCheckbox";
+import React from 'react';
+import FavouriteCheckbox from './FavouriteCheckbox';
 
-function FavouriteCheckboxContainer () {
-    return (
-        <FavouriteCheckbox />
-    );
-};
+function FavouriteCheckboxContainer() {
```

Comment:
no-op component, it is just wrapping another component. Please remove this and use the actual component directly.

Reply:
Deleted wrapped function